src/lib/helpers/notifications.ts

```diff
@@ -24,7 +24,7 @@ const notificationPrefs = (): Record<string, NotificationPrefItem> => {
     }
 
     // for some reason, the prefs become array as default or on all clear. let's reset.
-    return Array.isArray(prefs.notificationPrefs) ? {} : prefs.notificationPrefs || {};
+    return Array.isArray(prefs?.notificationPrefs) ? {} : (prefs?.notificationPrefs ?? {});
```

Comment:
There's already a null check above, though 🧐

Reply:
Ahhh it's a null check above...maybe that check should also check for undefined?